src/CMakeLists.txt

```diff
@@ -29,7 +29,7 @@ add_library(LLVMpta SHARED
 	llvm/analysis/PointsTo/Globals.cpp
 )
 
-target_link_libraries(LLVMpta PTA)
+target_link_libraries(LLVMpta PTA ${llvm_libs})
```

Comment:
> Could you try with target_link_libraries(LLVMpta PUBLIC PTA ${llvm_libs})?

That does not work, but this does:

```cmake
target_link_libraries(LLVMpta PUBLIC PTA
                              INTERFACE ${llvm_libs})
```


> BTW I get:
> 
> LLVMinstr 
> Usage: LLVMinstr <config.json> <llvm IR> <outputFileName>
>
> which is expected :)

when running `LLVMinstr` alone I get it too, the problem appears when you load the analysis (e.g. using memsafety.json)

Please check whether the modified `taget_link_libraries` works for you. If so, either add it to this PR or let me know and I'll merge it and follow up with this fix.

Thanks :)